client/src/components/assessments/AssessmentResultsTable.js

```diff
@@ -108,15 +112,27 @@ const AssessmentResultsTable = ({
   if (_isEmpty(periodsConfig?.periods)) {
     return null
   }
-  const entityInstantAssessmentConfig = entity.getInstantAssessmentConfig()
+  const entityInstantAssessmentConfig = Model.filterAssessmentConfig(
+    entity.getInstantAssessmentConfig(),
+    entity // TODO: add relatedObject?
```

Comment:
Left so we don't forget; I'm not 100% sure it is useful here, esp. regarding the aggregations.

Reply:
I have added extra condition for `relatedObject`, so that when we are aggregating the assessment questions, we don't filter `relatedObject` specific questions.